sandbox/optimal_args_hashbits.py

```diff
@@ -20,11 +20,11 @@
 import threading
 
 import khmer
-from khmer.khmer_args import build_hashbits_args
-from khmer.khmer_args import (report_on_config, info, add_threading_args)
+from khmer.khmer_args import (report_on_config, info, add_threading_args,
+                              build_hashbits_args)
 from khmer.kfile import check_input_files, check_space
 from khmer.kfile import check_space
-from oxli.functions import optimal_args_output_gen as output_gen
+from khmer.khmer_args import optimal_args_output_gen as output_gen
```

Comment:
in general, try to avoid renaming functions this way - makes finding them tricky.